examples/tests/linuxbuild.py

```diff
@@ -24,7 +24,8 @@ def setUp(self):
 
         self.linux_build = kernel.KernelBuild(kernel_version,
                                               linux_config,
-                                              self.srcdir)
+                                              self.srcdir,
+                                              ['/var/tmp'])
```

Comment:
Take a look at the patch before yours: https://github.com/ldoktor/avocado/commit/f6857f2b44e5119e71d1e37984b9fc05a9f0c5eb

As the `fetch_asset` uses that I think it's logical step to avoid hiding it and include it in the API. Alternatively we could include it in datadir. Well, that's probably even better place, what do you think?